### PR TITLE
Tutorial tweaks

### PR DESCRIPTION
**Pull request by wgranger**
Yet another fix after a PR review on the BPL repo. Should be fixed now.

https://github.com/zooniverse/liberating-the-liberator/pull/115

### CODE REVIEW

File: src/components/tutorial.jsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ if (window.tutorialsCompletedThisSession) {
 class Tutorial extends React.Component {
   componentWillMount() {
     this.previousActiveElement = document.activeElement;
+    this.handleNextClick = this.handleNextClick.bind(this);
```
Comment (srallen) [inline src/components/tutorial.jsx]: I think both of these could go into a class constructor (`componentWillMount` is likely going to be deprecated)

Comment (wgranger) [reply src/components/tutorial.jsx]: Oh neat. I never thought to make a function call in a class constructor.

Comment (wgranger) [reply src/components/tutorial.jsx]: Oh wait. Scratch that. I didn't notice when making this change that this was `componentWillMount` and not the constructor. Not firing on all cylinders today.

**Comment by wgranger** [comment]
I thought the same thing too but noticed the unmount was able to identify `this` correctly. Not sure if it's because it is executed as `this.handleUnmount()` vs `this.handleUnmount`? I can always throw in a binding just in case if you think it's necessary.

**Comment by eatyourgreens** [comment]
Binding is only required on event handlers, which change `this` to the context of the event, such as a clicked link. I think you're fine here.

**Comment by srallen** [comment]
Yes, you're right. I'm trying to multi-task too much...!

**Pull request merged**